style-guide.md

```diff
@@ -0,0 +1,585 @@
+save the table as a CSV file in a `/csv` directory within the appropriate
+chapter. For example, in the Administrator Manual's [Installation](https://github.com/artefactual/archivematica-docs/tree/1.8/admin-manual/installation-setup/installation)
+sub-chapter, there is a
+[_csv](https://github.com/artefactual/archivematica-docs/tree/1.8/admin-manual/installation-setup/installation/_csv)
+subdirectory that contains tables referenced in the pages of the Installation
+sub-chapter.
+
+To include a table on a page, reference the table using the `csv-table`
+directive.
+
+```
+.. csv-table::
+   :file: _csv/install-landing.csv
+   :header-rows: 1
+```
+
+Setting the `header-rows` option will make the header row a different colour
+than the body of the table.
+
+### Scripts
+
+Short, simple scripts can be added as code blocks using the `code` directive.
+
+```
+.. code:: bash
+
+   sudo wget -O - https://packages.archivematica.org/1.8.x/key.asc  | sudo apt-key add -
+   sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.8.x/ubuntu xenial main" >> /etc/apt/sources.list'
+   sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.8.x/ubuntu-externals xenial main" >> /etc/apt/sources.list'
+```
+
+For longer scripts, or scripts that will be referenced on multiple pages, you
+may wish to reference a script file, rather than inserting the script directly
+into the text.
+
+After creating and saving the script, place it in a `scripts` directory within
+the appropriate chapter. For example, in the Administrator Manual's [Installation](https://github.com/artefactual/archivematica-docs/tree/1.8/admin-manual/installation-setup/installation)
+sub-chapter, there is a [scripts](https://github.com/artefactual/archivematica-docs/tree/1.8/admin-manual/installation-setup/installation/scripts)
+subdirectory that contains scripts referenced in the pages of the Installation
+sub-chapter.
+
+To include a script on a page, reference the script using the `literal-include`
+directive.
+
+```
+.. literalinclude:: scripts/am18-centos-rpm.sh
+   :language: bash
+   :lines: 3
+```
+
+Setting the `language` option will display the script with appropriate syntax
+highlighting.
+
+Setting the `lines` option will display only the specified lines. You may
+specify one line, as above, or multiple lines:
+
+```
+.. literalinclude:: scripts/am18-centos-rpm.sh
+   :language: bash
+   :lines: 5-12
+```
+
+### Admonitions
+
+There are several different admonition directives available in reStructuredText.
+See the [reStructuredText Directives](http://docutils.sourceforge.net/docs/ref/rst/directives.html#figure)
+for a complete list.
+
+Commonly used in the Archivematica documentation are `note`, `important`, and
+`warning`. The chosen admonition will have an impact on how the contents are
+displayed.
+
+.. note::
+   Skip this step if you are planning to run :ref:`Archivematica without
+   Elasticsearch <install-elasticsearch>`.
```

Comment:
This could use a [fenced code block](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) since Markdown can't understand it.

```rst
.. note::
   Skip this step if you are planning to run :ref:`Archivematica without
   Elasticsearch <install-elasticsearch>`.
```

For the example above, I've written:

<pre>
```rst
.. note::
   Skip this step if you are planning to run :ref:`Archivematica without
   Elasticsearch <install-elasticsearch>`.
```
</pre>

Reply:
Hehe whoops!